src/OpenTelemetry/ProviderExtensions.cs

```diff
@@ -69,5 +72,18 @@ internal static Action GetObservableInstrumentCollectCallback(this BaseProvider
 
             return null;
         }
+
+        internal static Regex GetWildcardRegex(this BaseProvider baseProvider, IEnumerable<string> wildcards = default)
```

Comment:
This helper method doesn't have any specific logic to do with Providers. However, the providers are the only places that need it so I guess it's better to put it here than creating another place?

Reply:
👍 I agree, seems good for now.